analysis/rogueoutlaw/src/modules/features/Finishers.tsx

```diff
@@ -31,7 +31,7 @@ class Finishers extends FinisherTracker {
     return points;
   }
 
-  extraSuggestion() {
+  extraSuggestion(): React.ReactElement | any {
```

Comment:
how is `X | any` different from `any`? What type are you returning here that requires use of any?

Reply:
The issue is that `extraSuggestion()` here overrides `extraSuggestion()` from `FinisherTracker` in the base Rogue module that has not yet been converted  to Typescript. I can narrow this down to `React.ReactElement | string` by converting that module to Typescript.

Reply:
Leaving it as `any` is fine if you don't want to deal with in this PR, just leave a TODO comment

Reply:
Already did the conversion, it's a small module.